templates/cli/install.sh.twig

```diff
@@ -0,0 +1,187 @@
+<?php
+$version = $this->getParam('version', '');
+?>
```

Comment:
You can remove this, and just print the twig version param instead. The output of this file can't parse PHP code blocks.